NetExercise/Basic/Exercise17.cs

```diff
@@ -0,0 +1,44 @@
+namespace NetExercise.Basic
+{
+    public class Exercise17
+    {
+        public static int FindNumber(int n)
+        {
+            var num = n;
+
+            while(!(Prime(num) && SymmetricNumber(num)))
```

Comment:
Kết quả là như nhau nhưng viết thế này tốt hơn, anh đoán xem tại sao

```suggestion
            while(!(SymmetricNumber(num) && Prime(num)))
```

Reply:
vì hàm xữ lý SymmetricNumber sẽ chạy ít vòng hơn trong xử lý của hàm Prime(nếu số lớn tầm 100 là cũng gần 50 lần chạy để kiểm tra rồi 😊)

Reply:
Vì nguyên nhân là trong C# và nhiều ngôn ngữ khác, phép AND dạng a && b thì khi a đã sai thì b không cần tính toán nữa.

Do đó, nên chọn `a && b` hay `b && a` là xem cái nào chạy nhanh hơn, thường xuyên sai hơn thì để trước.